data/json/monsters/insect_spider.json

```
@@ -2021,5 +2021,400 @@
+  },
+  {
+    "id": "mon_antlion_larva",
+    "type": "MONSTER",
+    "name": { "str": "giant antlion larva" },
+    "description": "This huge grub has powerful mandibles and a hungry look.  It hides in the dirt and awaits its prey.",
+    "default_faction": "antlion_grub",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "80 L",
+    "weight": "106 kg",
+    "hp": 60,
+    "speed": 100,
+    "material": [ "iflesh" ],
+    "symbol": "u",
+    "color": "yellow",
+    "aggression": 80,
+    "morale": 100,
+    "melee_skill": 8,
+    "melee_dice": 1,
+    "melee_dice_sides": 8,
+    "melee_cut": 6,
+    "armor_bash": 4,
+    "armor_cut": 10,
+    "armor_bullet": 10,
+    "vision_day": 8,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "upgrades": { "age_grow": 12, "into": "mon_antlion_giant" },
+    "anger_triggers": [ "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "fear_triggers": [ "FIRE" ],
+    "special_attacks": [
+      [ "RANGED_PULL", 10 ],
+      [ "GRAB", 3 ],
+      [ "impale", 10 ],
+      {
+        "type": "bite",
+        "cooldown": 4,
+        "accuracy": 8,
+        "infection_chance": 75,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_penetration": 0.6 } ]
+      }
+    ],
+    "flags": [ "SEES", "SMELLS", "IMMOBILE", "DIGS", "RANGED_ATTACKER" ]
+  },
+  {
+    "id": "mon_antlion_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant adult antlion" },
+    "description": "A ferocious mutant flying antlion as big as a cat, darting through the air with a cluster of fangs for a mouth.  Despite dangerous looks it doesn't seem interested in you.",
+    "default_faction": "antlion_adult",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "volume": "30 L",
+    "weight": "40750 g",
+    "hp": 70,
+    "speed": 130,
+    "material": [ "iflesh" ],
+    "symbol": "y",
+    "color": "yellow",
+    "aggression": -10,
+    "morale": 20,
+    "melee_skill": 6,
+    "melee_dice": 1,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 8,
+    "armor_cut": 4,
+    "vision_day": 25,
+    "vision_night": 5,
+    "harvest": "arachnid_flying",
+    "reproduction": { "baby_egg": "egg_antlion", "baby_count": 1, "baby_timer": 15 },
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN" ],
+    "anger_triggers": [ "HURT" ],
+    "fear_triggers": [ "PLAYER_CLOSE", "FIRE" ],
+    "flags": [ "SEES", "SMELLS", "FLIES", "HEARS", "HIT_AND_RUN", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_strider_small",
+    "type": "MONSTER",
+    "name": { "str": "big water strider" },
+    "description": "This water bug is several times larger than a regular one.  It's moves are fast and fluid.",
+    "copy-from": "mon_strider_giant",
+    "proportional": { "hp": 0.5, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.5 },
+    "volume": "3 L",
+    "weight": "4 kg",
+    "melee_cut": 8,
```

```suggestion
    "melee_cut": 1,
```
Small babies shouldn't be cuttier than the adults.